src/admin-views/editor/fieldset/price.php

```diff
@@ -54,11 +54,26 @@ class="price tribe-dependent"
 			class="ticket_field ticket_form_right"
 			size="7"
 			value="<?php echo esc_attr( $ticket ? $price : null ); ?>"
-			<?php echo implode( ' ', $validation_attrs ) ?>
+			<?php echo implode( ' ', $validation_attrs ); ?>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: ec175fd06cbd8a47e45ca84799443f8d</sub>